scripts/train/benchmarking/sweep.sh

```diff
@@ -1,49 +1,61 @@
 #!/bin/bash
 
-PROJECT="tput"
+PROJECT="opt30"
 GIT_COMMIT="v0.0.4"
-IMAGE="mosaicml/pytorch:1.13.1_cu117-python3.10-ubuntu20.04"
-CLUSTER_80GB=YOUR_CLUSTER_80GB
-CLUSTER_40GB=YOUR_CLUSTER_40GB
+IMAGE="mosaicml/pytorch:2.0.1_cu118-python3.10-ubuntu20.04"
+# IMAGE="mosaicml/pytorch:2.1.0_cu121-nightly20230827-python3.10-ubuntu20.04"
```

Comment:
there is probably a newer nightly

Reply:
This is fine. Let's use this nightly for now. We will update to a stable pin when torch drops new release in 1-2 days